src/main/java/core/basesyntax/utils/generate/impl/csv/CsvGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.utils.generate.impl.csv;
+
+import core.basesyntax.storage.impl.StorageImpl;
```

Comment:
Consider removing the direct dependency on `StorageImpl`. Instead, use the `Storage` interface to adhere to the Dependency Inversion principle of SOLID. This will make your code more flexible and easier to test.